src/switches.js

```diff
@@ -20,15 +20,25 @@ chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
   const currentUrl = tabs[0].url;
   const url = new URL(currentUrl);
   const host = url.hostname;
-
-  const social = host === "www.linkedin.com" ? "linkedin" : "facebook";
-
+  const social = (host)=> {
```

Comment:
May I ask you to refactor this as a `switch/case` statement?
It will be easier and cleaner to extend, and we also can fall through multiple `case` statements for situations such as line 28 (`host === "www.twitter.com" || host === "twitter.com"`)